docs/389ds/FAQ/faq.md

```diff
@@ -117,25 +117,25 @@ See [History](../FAQ/history.html) for a history of Netscape, iPlanet, and Sun D
 
 Since the break up of iPlanet, the products have followed different development paths. However, much of the behavior and configuration are still the same: the configuration backend, schema, monitoring, management tools, etc. There are a couple of significant differences which you need to be aware of. Sun DS 5.2 changed the replication protocol, so you cannot do Sun DS 5.2 style replication with 389. However, Sun DS 5.2 has a legacy replication mode with which they are able to replicate to their 5.1 and 5.0 servers. This replication mode works with 389. You can use Sun DS 5.2 as a replication hub - it can replicate with newer versions of Sun/Oracle DS on one side, and replicate with RHDS on the other side. Also, Sun DS 5.2 has a different database format, so database files cannot be shared between the two servers. You must export your data to LDIF then import into 389. There are several other differences but they mostly don't affect interoperability. Please refer to the Sun DS documentation for more information.
 
-### How does the 389 multi-master replication work?
+### How does the 389 multi-supplier replication work?
 
-389 replication uses a push model. A replication supplier is a server that pushes updates to other servers. A master is a supplier that can receive update requests from clients. A hub is a supplier that receives update requests from other replication suppliers and pushes updates to other replicas.
+389 replication uses a push model. A replication supplier is a server that pushes updates to other servers. A supplier is a supplier that can receive update requests from clients. A hub is a supplier that receives update requests from other replication suppliers and pushes updates to other replicas.
```

Comment:
Would change it into: A supplier is a LDAP server that can receive and process update requests from clients.